.github/workflows/dockerimage.yml

```diff
@@ -64,6 +64,10 @@ jobs:
       env:
         DOCKER_BUILDKIT: 1
 
+    - name: Output installed packages
+      run: |
+        pip freeze --all
```

Comment:
This command doesn't run in the container. I think replacing 
```
pip freeze --all
```
with
```
docker run "${HUB_TAG}" sh -c "pip freeze --all"
```
would be one of the possible way to see the dependency in the container.